docker/build.sh

```diff
@@ -45,6 +49,12 @@ shift 1
 DOCKERFILE_PATH="${SCRIPT_DIR}/Dockerfile.${CONTAINER_TYPE}"
 DOCKER_CONTEXT_PATH="${SCRIPT_DIR}"
 
+if [[ "$1" == "--dockertag" ]]; then
```

Comment:
not sure the "docker" prefix is necessary, since this script is already `docker/build.sh`.

Reply:
Agreed. I'll remove.